packages/nimble-components/src/select/index.ts

```diff
@@ -49,6 +69,120 @@ export class Select extends FoundationSelect implements ErrorPattern {
     @observable
     public hasOverflow = false;
 
+    /**
+     * @internal
+     */
+    @observable
+    public filteredOptions: ListboxOption[] = [];
+
+    /**
+     * @internal
+     */
+    @observable
+    public filter = '';
+
+    /**
+     * @internal
+     */
+    @observable
+    public scrollbarIsVisible = false;
+
+    private committedSelectedOption: ListboxOption | undefined = undefined;
+
+    public constructor() {
+        super();
+        this.addEventListener('change', this.changeValueHandler);
+        this.addEventListener('focusout', this.focusoutHander);
+    }
+
+    /**
+     * The list of options. This mirrors FAST's override implementation for this
+     * member for the Combobox to support a filtered list in the dropdown.
+     *
+     * @public
+     * @remarks
+     * Overrides `Listbox.options`.
+     */
+    public override get options(): ListboxOption[] {
+        Observable.track(this, 'options');
+        return this.filteredOptions?.length
+            ? this.filteredOptions
+            : (this._options as ListOption[]);
+    }
+
+    public override set options(value: ListboxOption[]) {
+        this._options = value;
+        this.committedSelectedOption = value.find(option => option.selected);
+        Observable.notify(this, 'options');
+    }
+
+    // See comment for 'set value' below. Despite the implementation for 'get value'
+    // being an exact copy of the FAST Select one, without replicating it here, we
+    // would see test errors.
```

Comment:
Basically tests that would access the current `value` to check to see if it was the expected value would fail. At the moment I'm not certain of the reason why having the local implementation is necessary.

Reply:
I think it would be good to understand why. I wonder if it has to do with the `this` value that ends up being passed to `Observable.track(this, 'value');`.

Reply:
It was discovered that per the spec, if you provide a setter implementation, you must also provide a getter to avoid a return value of `undefined` (see [this SO post](https://stackoverflow.com/questions/28950760/override-a-setter-and-the-getter-must-also-be-overridden)).